OWNERS

```diff
@@ -0,0 +1,19 @@
+approvers:
+- calebevans
+- btjd
+- ascerra
+- myakove
+- rnetser
+options: {}
```

Comment:
Why is `options: {}` here for?

Reply:
I think it's just a carryover. I'll remove it